fw/t/fuzzer.c

```diff
@@ -453,24 +524,55 @@ add_field(TfwFuzzContext *ctx, int type, char **p, char *end, int t)
 static unsigned int
 __add_header(TfwFuzzContext *ctx, int type, char **p, char *end, int t, int n)
 {
+	static char *saved_p = NULL, *saved_end = NULL;
+	static char h2_buf[10*1024*1024] = {};
+	static size_t remain_p = 0;
 	unsigned int v = 0, i;
 
 	BUG_ON(t < 0);
 	BUG_ON(t >= TRANSFER_ENCODING_NUM);
 
-	BUG_ON(!fld_data[t].key);
+	BUG_ON(!fld_data[t].h1_key && !fld_data[t].h2_key);
+
+	if (type == FUZZ_REQ_H2) {
+		saved_p = *p;
+		saved_end = end;
+		remain_p = saved_end - saved_p;
+		BUG_ON(remain_p > sizeof(h2_buf));
+		*p = h2_buf;
+		end = h2_buf + remain_p;
+	}
+	else
+	{
```

Comment:
```suggestion
	} else {
```
also see the coding style

Reply:
Done